java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -595,6 +597,30 @@ public CompletableFuture<Long> zcard(@NonNull String key) {
         return commandManager.submitNewCommand(Zcard, new String[] {key}, this::handleLongResponse);
     }
 
+    @Override
+    public CompletableFuture<String> zrandmember(@NonNull String key) {
+        return commandManager.submitNewCommand(
+                ZRandMember, new String[] {key}, this::handleStringOrNullResponse);
+    }
+
+    @Override
+    public CompletableFuture<String[]> zrandmemberWithCount(@NonNull String key, long count) {
+        return commandManager.submitNewCommand(
+                ZRandMember,
+                new String[] {key, Long.toString(count)},
+                response -> castArray(handleArrayResponse(response), String.class));
+    }
+
+    @Override
+    public CompletableFuture<Object[][]> zrandmemberWithCountWithScores(
+            @NonNull String key, long count) {
+        String[] arguments = new String[] {key, Long.toString(count), "WITHSCORES"};
+        return commandManager.submitNewCommand(
+                ZRandMember,
+                arguments,
+                response -> castOneDimensionalArrayToTwoDimensional(handleArrayResponse(response)));
```

Comment:
```suggestion
                response -> castArray(handleArrayResponse(response), Object[].class));
```
and remove the new function

Reply:
Didn't think of that cool!